std/math.d

```diff
@@ -949,7 +952,8 @@ Lret: {}
         z = ((x - y * P1) - y * P2) - y * P3;
         const real zz = z * z;
 
-        if (zz > 1.0e-20L)
+        enum zzThreshold = (floatTraits!real.realFormat == RealFormat.ieeeDouble ? 1.0e-14L : 1.0e-20L);
```

Comment:
Store `floatTraits!real` in an alias instead of instantiating it multiple times in the same function?

Reply:
Twice to be precise, and an `enum isRealDouble` would suffice. If you insist on adding a new line for shortening these 2 lines, I won't refuse. ;)

Reply:
There was a third usage I previously overlooked => `enum realFormat`.